kategory/src/main/kotlin/kategory/data/NonEmptyList.kt

```diff
@@ -60,7 +60,11 @@ package kategory
         fun <A> semigroup(): Semigroup<NonEmptyList<A>> = object : NonEmptyListSemigroup<A> {}
 
         fun semigroupK(): SemigroupK<NonEmptyListHK> = object : NonEmptyListSemigroupK {}
+
+        fun foldable(): Foldable<NonEmptyListHK> = this
+
+        fun traverse(): Traverse<NonEmptyListHK> = this
     }
 }
 
-fun <A> A.nel(): NonEmptyList<A> = NonEmptyList.of(this)
+fun <A> A.k(): NonEmptyList<A> = NonEmptyList.of(this)
```

Comment:
I'd personally revert this change. k() is reserved for kind wrappers for list or function. In this case you're creating a nel() from a, the same way we have left() and right() for Either, success() for Try or valid() for Validated.